services/explorer/src/models/TableData.ts

```diff
@@ -1,6 +1,6 @@
 export enum TransactionStatus {
-  SUCCESS = '0x0',
-  FAILURE = '0x1',
+  SUCCESS = '0x1',
```

Comment:
This is not really used to be honest. I saw that the status is either 0, 1 or null. I created a new method to convert this (formatTransactionStatus).
Just double check if with this change, the status show the correct status (I think it was a component in taraxa-ui)

Reply:
Yeah, it was switched-up beforehand. Should be fine now.